### DISCUSSION TITLE
unable to download templates <- context deadline exceeded

### DISCUSSION THREAD

**Discussion by zachluh** [projectdiscovery/nuclei-templates#11008] [Q&A]
![image](https://github.com/user-attachments/assets/2eb1b3fa-8c6e-452f-8c09-67dbf5d682f1)

I'm trying to run nuclei and I keep getting this message. I checked the nuclei-templates file after and it was completely empty. This could be somehow happening due to the fact I previously installed nuclei with `sudo apt install nuclei` instead of using the github link. How can I fix this?

**Comment by PatrikFehrenbach** [comment]
apt-get remove nuclei / apt remove nuclei ;-)
